autoinstall/README.md

```diff
@@ -18,6 +18,7 @@
 17. For UnionTech OS Server 20 1050e unattend auto-install, please use file UOS/Server/20/1050e/ks.cfg.
 18. For Fedora Server 36 or later unattend auto-install, please use file Fedora/36/Server/ks.cfg.
 19. For FreeBSD 13 or later unattend auto-install, please use file FreeBSD/installerconfig.
+20. For Pardus 21.x or later unattend auto-install, please use file Pardus/preseed.cfg.
```

Comment:
21.x means from 21.0. So please test your fix with Pardus 21.0. Or you specify the version started to support. I saw Pardus has 3 editions: GNOME, Xfce, and Server. Does this preseed.cfg supports all of the 3 editions?

Reply:
Change it. Thanks